README.md

```diff
@@ -97,7 +97,22 @@ sourceTypes: # these are the data source types that are supported to upload data
       - "application/zip"
     timeRequired: false
     sourceIdRequired: false
-
+  - name: oxford-wearable-camera
+    topics:
+      - connect_upload_oxford_camera_data
+      - connect_upload_oxford_camera_image
+    contentTypes:
+      - application/zip
+    # requires SFTP server config to store images to
+    configuration:
+      host: sftp  # sftp host to upload images to
+      # port: 22  # sftp port already defaults to 22
+      user: ...  # sftp username
+      # password: ...  # sftp password. Prefer to use the keyfile
+      keyFile: </path/to/private/ssh/key>  # path to private SSH key. Can be replaced with password if needed.
```

Comment:
do we have to add this path to the volumes of this container when deployed?

Reply:
Yes, either a mounted key or the password should be used.